### PR TITLE
Add 5 icons

### PR DESCRIPTION
**Pull request by mcellier**
We should really do #221 someday.

### CODE REVIEW

**Comment by Bidjii** [comment]
Don't forget to camelCase html attributes like `stroke-linecap` > `strokeLinecap`
You'll get warnings otherwise

**Comment by mcellier** [comment]
Note: each icon has been optimized through https://jakearchibald.github.io/svgomg/

File: src/icon/avatarIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    iconColor: color.icon,
+    size: 24,
+    title: '',
+  }
+
+  render() {
+    const { className, iconColor, size, title } = this.props
+    return (
+      <svg
+        viewBox="0 0 19 20"
```
Comment (wakooka) [inline src/icon/avatarIcon.tsx]: viewbox 19?

**Comment by mcellier** [comment]
> Don't forget to camelCase html attributes like `stroke-linecap` > `strokeLinecap`
> You'll get warnings otherwise

Done. Thx.

File: src/icon/avatarIcon.tsx (hunk 1 above)
Comment (mcellier) [reply src/icon/avatarIcon.tsx]: All these icons have different sizes... We already have some specificities like this :(

Comment (mcellier) [reply src/icon/avatarIcon.tsx]: Search `viewBox=` and see the thruth

File: src/icon/avatarIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        xmlns="http://www.w3.org/2000/svg"
+        className={cc(['kirk-icon', className])}
+        width={size}
+        height={size}
+        aria-hidden={isEmpty(title)}
+        fill={iconColor}
+      >
+        {title && <title>{title}</title>}
+        <g fill="none">
+          <path d="M-2-2h24v24H-2z"/>
```
Comment (Bidjii) [inline src/icon/avatarIcon.tsx]: What is this path used for? I noticed you have one on each icon and it seems that it's useless.

**Pull request merged**